1-exercises/D-do-while/exercise.js

```diff
@@ -1,15 +1,44 @@
 /*
-    Sometimes when using loops, we'll want to execute the body of the loop at least once. We can make sure this happens by using a do-while loop.
+    Sometimes when using loops, we'll want to execute the body of the loop at least once. 
+    We can make sure this happens by using a do-while loop.
         - If the condition in a while loop is initially false, the body of the loop will never execute
-        - But in a do-while loop, because the condition is checked after the body, we know that it will always execute at least once
+        - But in a do-while loop, because the condition is checked after the body, 
+          we know that it will always execute at least once
     
-    Using a do-while loop, write a function which returns the sum of the first n even numbers (starting from 0)
+    1 Using a do-while loop, 
+    2 write a function which returns 
+      *the sum of 
+      *the first n even numbers (starting from 0)
 */
 
 function evenNumbersSum(n) {
     // TODO
+   
+    let sum = 0;
+     let arr = [];
+    for (let i = 0; i < n; i++){
+        if (i >0 && i % 2 === 0) {
+            arr.push(i);
+        };
```

Comment:
This is decent code but it doesn't "return the sum of the first n even numbers (starting from 0)"